components/definitions/ethereum-blockies.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'ethereum-blockies'
```

Comment:
What's the point of this if it's empty? Can we add the basic API signature anyway?

Reply:
The reason I added this was to get rid of this type declaration error.

![Screen Shot 2021-04-16 at 6 50 04 PM](https://user-images.githubusercontent.com/40611140/115096943-a7360700-9ee4-11eb-8c78-06b4a37ec137.png)